### PR TITLE
[Enahance] Update variants fields defined in the Item Variant Settings, if template updated

### LINKED ISSUE #9482: Variants not Updated when Item Master is Updated
If Item Variants are not updated when you update Item Template there is no need to use Variants anymore.
Are you willing to reintroduce this function? Why did you remove it?
I chose ERPNext instead of Odoo also because I preferred the way variants were managed.
Last I think you should let also users of Hosted Plan to be able to decide which fields can be set as NoCopy.

### PR DESCRIPTION
**Pull request by rohitwaghchaure**
- [x] Default values on Patch + Install
- [x] Rename "Variant Fields" to "Copy Fields to Variant"
---
**Issue**
Variants data not updated if user update the data in the temaplte

**Fixed**
Select the fields in the  Item Variant Settings which want to update in the variants master from the template
screen shot 2017-09-11 at 12 44 00 pm

Fixed https://github.com/frappe/erpnext/issues/9482

### CODE REVIEW

**Comment by GiorgioKN** [comment]
Hi,
thank you very good news.
Could you please tell me when this will be available for the cloud version?

**Comment by rohitwaghchaure** [comment]
Hi @GiorgioKN 

Currently it's under review and we'll release it once PR is merged!

File: erpnext/config/stock.py [hunk 1]
```diff
@@ -73,6 +73,11 @@ def get_data():
 				},
 				{
 					"type": "doctype",
+					"name": "Item Variant Settings",
```
Comment (rmehta) [inline erpnext/config/stock.py]: this should be in settings!

**Review by rmehta — CHANGES_REQUESTED** [review]

File: erpnext/config/stock.py (hunk 1 above)
Comment (rmehta) [reply erpnext/config/stock.py]: also maybe add a button in Item (if it is a variant) for Variant Settings

**Comment by rmehta** [comment]
patch + default settings on install.

**Review by rmehta — APPROVED** [review]

**Comment by nabinhait** [comment]
merged from console

**Pull request closed without merging**